### ISSUE TITLE
interpolations in routed <ng-content>  only resolves after timeout (!)

### ISSUE THREAD

**Issue by lmessinger** [angular/angular#4192]
tried to use this simple approach on an application with a router, and it turned out not so simple.

one issue, is that with a router, the transcluded ng-content is not bound at first. 

But the really interesting issue, is that after setTimeout, the items are bound again. 

The plunker is here: http://plnkr.co/edit/gGZJZE?p=preview. click on Entities to see issue

thanks for any explanation/resolution

**Comment by Necroskillz** [comment]
:+1: i also came across this issue. in my case it's a tabs component, which doesn't work optimally with routing.

I made this example before I found this issue, so might as well add it here: http://plnkr.co/edit/XkYGcaAi6gWgzR1ucmaO?p=preview - switch to `other route` then `Tabs demo` - the content of `Dynamic tabs` is not visible until some change happens.

This is a fairly simple component, but uses `ng-for` to render items, that are displayed with `ng-content` and the items also use `ng-content` themselves.
